assets/package.json

```diff
@@ -6,13 +6,12 @@
   "scripts": {
     "build": "webpack --config webpack.prod.js",
     "watch": "webpack --watch --config webpack.dev.js",
-    "pretest": "eslint src",
-    "pretest-fix": "eslint --fix src"
+    "pretest": "npx eslint src --ignore-pattern src/legacy",
+    "pretest-fix": "npx eslint --fix src --ignore-pattern src/legacy"
```

Comment:
I think this change is linked to my previous question https://github.com/3liz/lizmap-web-client/pull/5294#discussion_r1934263498

As discussed this morning, it would make more sense to merge all `package.json` at the root, to have these `scripts` shared among all the source code (tests included). And playwright and jsunit should be merge in the `devDependancies` instead in the core part.

CC @nboisteault